tblgen/hs-generators.cc

```diff
@@ -468,8 +471,9 @@ void emitBuilderMethod(mlir::tblgen::Operator& op,
   std::vector<std::string> operand_binders;
   std::vector<std::string> operand_name_exprs;
   operand_name_exprs.reserve(op.getNumOperands());
-  for (const auto& operand : op.getOperands()) {
-    std::string operand_name = sanitizeName(operand.name);
+  for (int i = 0; i < op.getNumOperands(); ++i) {
```

Comment:
Nit: preference towards 

for (int i =0, e = op.getNumOperands(); i < e; ++i)

to document that array size doesn't change. Alternatively, llvm::enumerate is useful for this too

Reply:
I don't want to use C++17 and without structured bindings it's a bit of a pain. I'll keep it as is. I don't think we are at a point where we want to hyper-optimize this C++ code and it is a bit cleaner as is.